src/managed/dir.proj

```diff
@@ -5,12 +5,16 @@
 
   <UsingTask TaskName="GetBuildArgsByFrameworks" AssemblyFile="$(LocalBuildToolsTaskDir)core-setup.tasks.dll"/>
 
-  <Target Name="Build">
+  <Target Name="Build"
+          DependsOnTargets="GetLatestCommitHash">
     
     <MakeDir  Condition="!Exists('$(pOutDir)')" Directories="$(pOutDir)" /> 
 
     <PropertyGroup>
       <Args>/p:Configuration=$(ConfigurationGroup)</Args>
+      <_EnvironmentVariables>NUGET_PACKAGES=$(PackagesDir)</_EnvironmentVariables>
+      <_EnvironmentVariables Condition="'$(LatestCommit)' != ''">$(_EnvironmentVariables);LatestCommit=$(LatestCommit)</_EnvironmentVariables>
+      <_EnvironmentVariables Condition="'$(BuiltByString)' != ''">$(_EnvironmentVariables);BuiltByString=$(BuiltByString)</_EnvironmentVariables>
```

Comment:
It's in the same buildversion.props file which gets generated and includes `BuildNumberMajor` and `BuildNumberMinor`